docs/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml

```diff
@@ -3220,111 +3220,40 @@ split_words('Once upon a time');
+        within The HPCC Systems<superscript>®</superscript> Platform:</para>
 
         <orderedlist>
           <listitem>
-            <para>Install R-core distribution of your choice:</para>
-
-            <para><emphasis role="bold">RPM-based systems:</emphasis></para>
-
-            <para>On an RPM-based system (CentOS/Red Hat) use <emphasis
-            role="bold">R-core</emphasis> and <emphasis
-            role="bold">R-core-devel</emphasis></para>
+            <para>Download the <emphasis role="bold">R Embed</emphasis> plugin
+            available from the HPCC Systems portal.</para>
 
-            <para><emphasis role="bold">Debian-based
-            systems:</emphasis></para>
+            <para><ulink
+            url="https://hpccsystems.com/download/hpcc-platform">https://hpccsystems.com/download/hpcc-platform</ulink></para>
 
-            <para>For a Debian-based (Ubuntu) system use <emphasis
-            role="bold">r-base-core</emphasis>.</para>
+            <para>Choose the appropriate plugin for your distribution.</para>
           </listitem>
 
           <listitem>
-            <para>Install the <emphasis role="bold">Rcpp</emphasis> and the
-            <emphasis role="bold">RInside</emphasis> libraries.</para>
-
-            <para>You will need both Rcpp and RInside libraries in order for
-            the R-embedding to work properly. The library installation
-            packages are named with the version number appended so you should
-            search for Rcpp_ and RInside_ to obtain the latest version for
-            your system.</para>
-
-            <para><emphasis role="bold">For all distros except Ubuntu
-            14.04:</emphasis></para>
-
-            <programlisting>wget http://cran.r-project.org/src/contrib/<emphasis
-                role="bold">Rcpp_0.10.4.</emphasis>tar.gz
-wget http://cran.r-project.org/src/contrib/<emphasis role="bold">RInside</emphasis><emphasis
-                role="bold">_0.2.10.tar.gz</emphasis>
-</programlisting>
-
-            <para>To Install:</para>
-
-            <programlisting>sudo R CMD INSTALL <emphasis role="bold">Rcpp_0.10.4.</emphasis>tar.gz
-sudo R CMD INSTALL <emphasis role="bold">RInside_0.2.10</emphasis>.tar.gz</programlisting>
-
-            <para><emphasis role="bold">For Ubuntu 14.04:</emphasis></para>
-
-            <programlisting>wget http://cran.r-project.org/src/contrib/<emphasis
-                role="bold">Rcpp_0.11.0.</emphasis>tar.gz
-</programlisting>
+            <para>Install the plugin.</para>
 
-            <para>To Install:</para>
+            <para><emphasis role="bold">RPM-based systems: </emphasis>Install
+            using <emphasis>yum install</emphasis>.</para>
 
-            <programlisting>sudo R CMD INSTALL <emphasis role="bold">Rcpp_0.11.0.</emphasis>tar.gz
-</programlisting>
+            <para><emphasis role="bold">Debian-based systems:
+            </emphasis>Install using <emphasis>apt-get install
```

Comment:
For debian you might want to say "Install using <emphasis>dpkg -i</emphasis> followed by <emphasis>apt-get install -f</emphasis>.